static-server/app.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const path = require("path");
+const logger = require("morgan");
+const fs = require("fs");
+const cors = require("cors");
+
+const app = express();
+
+app.use(cors());
+app.use(logger("dev"));
+app.use(express.static(path.join(__dirname, "../build")));
```

Comment:
What is this directory?

Reply:
This is the build directory for the example-client. It's not necessary if you configure the example client to point at the store. I'll take it out for simplicity's sake.